examples/sandbox/package.json

```diff
@@ -5,13 +5,14 @@
   "private": true,
   "browserslist": "> 0.5%, last 2 versions, not dead",
   "scripts": {
-    "build": "rm -rf public; yarn parcel build index.html --dist-dir public",
-    "dev": "yarn parcel index.html",
-    "clean": "rm -rf dist node_modules public .parcel-cache"
+    "build": "rm -rf public && vite build --outDir public",
+    "preview": "vite",
+    "clean": "rm -rf dist node_modules public"
   },
   "dependencies": {
     "@metamask/eth-sig-util": "^7.0.0",
     "@shapeshiftoss/hdwallet-core": "1.55.2",
+    "@shapeshiftoss/hdwallet-keepkey": "1.55.2",
```

Comment:
sandbox is derped and it working was accidental due to how parcel resolved modules, it was actually missing a ton and would stderr on build